src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,368 @@
+# Imaging data types
+
+This section pertains to imaging data, which characteristically have spatial
+extent and resolution.
+
+## Preprocessed, coregistered and/or resampled volumes
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<ext>
+```
+
+Volumetric preprocessing does not modify the number of dimensions, and so
+the specifications in [Preprocessed or cleaned data][common_preproc]
+apply.
+The use of surface meshes and volumetric measures sampled to those meshes is
+sufficiently similar in practice to treat them equivalently.
+
+When two or more instances of a given derivative are provided with resolution
+or surface sampling density being the only difference between them, then the
+`res` (for *resolution* of regularly sampled N-D data) and/or `den` (for
+*density* of non-parametric surfaces) SHOULD be used to avoid name conflicts.
+Note that only files combining both regularly sampled (e.g., gridded) and surface
+sampled data (and their downstream derivatives) are allowed to present both `res`
+and `den` keywords simultaneously.
```

Comment:
`res` and `den` are not "keywords" but "entities" as defined per https://bids-specification.readthedocs.io/en/common-derivatives/02-common-principles.html#file-name-structure.